src/test/java/org/broadinstitute/consent/http/service/DataAccessReportsParserTest.java

```diff
@@ -77,41 +78,44 @@ void testDataAccessApprovedReport() throws IOException {
     parser.addApprovedDARLine(darWriter, election, dar, DAR_CODE, REQUESTER, ORGANIZATION,
         CONSENT_NAME, sDUL);
     darWriter.flush();
-    Stream<String> stream = Files.lines(Paths.get(file.getPath()));
-    Iterator<String> iterator = stream.iterator();
-    int i = 0;
-    while (iterator.hasNext()) {
-      String line = iterator.next();
-      String[] columns = line.split("\t");
-      assertEquals(12, columns.length);
-      if (i == 0) {
-        assertEquals(columns[0], HeaderDAR.DAR_ID.getValue());
-        assertEquals(columns[1], HeaderDAR.DATASET_NAME.getValue());
-        assertEquals(columns[2], HeaderDAR.DATASET_ID.getValue());
-        assertEquals(columns[3], HeaderDAR.CONSENT_ID.getValue());
-        assertEquals(columns[4], HeaderDAR.DATA_REQUESTER_NAME.getValue());
-        assertEquals(columns[5], HeaderDAR.ORGANIZATION.getValue());
-        assertEquals(columns[6], HeaderDAR.CODED_VERSION_SDUL.getValue());
-        assertEquals(columns[7], HeaderDAR.CODED_VERSION_DAR.getValue());
-        assertEquals(columns[8], HeaderDAR.RESEARCH_PURPOSE.getValue());
-        assertEquals(columns[9], HeaderDAR.DATE_REQUEST_SUBMISSION.getValue());
-        assertEquals(columns[10], HeaderDAR.DATE_REQUEST_APPROVAL.getValue());
-        assertEquals(columns[11],
-            HeaderDAR.DATE_REQUEST_RE_ATTESTATION.getValue());
+    try (Stream<String> stream = Files.lines(Paths.get(file.getPath()))) {
```

Comment:
with-resources!

Reply:
Yes, this is a real problem in current tests. I've been tackling them on an as-needed basis but that's been very slow.

Reply:
Often `try-with-resources` is a great place to use `var`
```suggestion
    try (var stream = Files.lines(Paths.get(file.getPath()))) {
```

Although, from looking at this code, the input file is fixed length and only has two lines, so using a `Stream` is unnecessary.
```
    var lines = Files.readAllLines(Paths.get(file.getPath()))
```